crates/bevy_asset/src/folder.rs

```diff
@@ -5,6 +5,8 @@ use bevy_reflect::TypePath;
 
 /// A "loaded folder" containing handles for all assets stored in a given [`AssetPath`].
 ///
+/// This is produced by [`AssetServer::load_folder`](crate::prelude::AssetServer::load_folder).
```

Comment:
For my own learning, do these links work? I thought you either need it to guess it automatically or there's a "symbol" section in another paragraph.